pypeit/wavecalib.py

```diff
@@ -705,7 +711,11 @@ def build_wv_calib(self, arccen, method, skip_QA=False,
                     self.meta_dict['dispname'],
                     angle_fits_file,
                     composite_arc_file,
-                    pad=3, debug=False)
+                    pad=self.par['echelle_pad'],
+                    cc_percent_ceil = self.par['cc_percent_ceil'], debug=False)
+            # Padding should be a spectrograph-specific, or calibration-mode-specific quantity.
```

Comment:
Remove these comments since this was implemented?

Reply:
removed.